index.html

```diff
@@ -305,6 +305,13 @@ <h1>YouTubers' Birthdays</h1>
 					<td>Janurary 17, 1995</td>
 					<td>147 million</td>
 				</tr>
+		<!-- coding -->
+				<tr>
+					<td>Web development</td>
```

Comment:
Should we say Web Development or Tech? What do you think @Gallantfarhan?

Reply:
Web Development sounds perfect!